.github/workflows/run-acceptance-tests.yml

```diff
@@ -388,6 +389,10 @@ jobs:
       with:
         token: ${{ secrets.GITHUB_TOKEN }}
         version: v2.4.0
+    - name: Setup SSH key
+      uses: webfactory/ssh-agent@v0.7.0
+      with:
+        ssh-private-key: ${{ secrets.PRIVATE_SSH_KEY_FOR_DIGITALOCEAN }}
```

Comment:
Won't this be overriden by ci-mgmt?

Reply:
True. Addressed by pulumi/ci-mgmt#517.

Reply:
I believe this part will still be overriden. All #517 does is set `DIGITALOCEAN_TOKEN`.